### PR TITLE
Replace the MIT license with GPL.

### PR DESCRIPTION
**Pull request by GoldenBadger**
If approved, all code on the repo after that point will be licensed
under the GPLv3.

### CODE REVIEW

**Comment by butlerx** [comment]
Im not a big fan of replacing the license i prefer bsd

**Comment by CSmartt** [comment]
Approved by admins, webmaster & chair, merging this soon unless someone has a good reason not to.

**Comment by butlerx** [comment]
I still haven't heard a reason, just nods of approval, I'm wondering were a move to copyleft is coming from as redbrick is normally unlicenced or mit

File: LICENSE.md [hunk 1]
```diff
@@ -1,19 +0,0 @@
-The MIT License (MIT)
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
```
Comment (butlerx) [inline LICENSE.md]: Redbrick doesn't hold the copyright to the code therefor cant re-licence can only sub-licence the code

Comment (CSmartt) [reply LICENSE.md]: According to the current license, nor does anyone else, to my knowledge.

**Comment by GoldenBadger** [comment]
It seems we were not using the MIT license correctly anyway, as we do not include a copyright notice.

https://tldrlegal.com/license/mit-license

File: LICENSE.md (hunk 1 above)
Comment (butlerx) [reply LICENSE.md]: if it isn't stated the copyright is help by all contributors

Comment (CSmartt) [reply LICENSE.md]: As far as I can see, MIT requires an explicit declaration of the copyright holder, I'm open to correction here however.

Comment (butlerx) [reply LICENSE.md]: without an explicit declaration, copyright is not assigned 
and all contributor implicitly own the copyright on their own code

**Comment by GoldenBadger** [comment]
Note, we can change the copyright notice to something like `Copyright (C) 2017 the official Redbrick website contributors`.

File: README.md [hunk 2]
```diff
@@ -111,3 +111,22 @@ variables.
 | `SMTP_PASS`            |                            | account password                                                 |
 | `SMTP_AUTH_METHOD`     | `plain`                    | auth method to use with SMTP. Defaults to plain                  |
 | `PORT`                 | `3000`                     | port to run the server on defaults to 3000                       |
+
+## License
+
+Official website of DCU Redbrick Society.
+Copyright (C) 2017 Redbrick
```
Comment (butlerx) [inline README.md]: redbrick doesn't own the copyright

Comment (GoldenBadger) [reply README.md]: Can you explain?

**Comment by butlerx** [comment]
the definition of official is ambiguous

**Comment by GoldenBadger** [comment]
What do you recommend we change it to? It is the officially-supported website.

**Comment by butlerx** [comment]
reading of it seems you are saying official contributors rather then contributors to the official website

**Comment by GoldenBadger** [comment]
I will reword.

File: README.md [hunk 3]
```diff
@@ -111,3 +111,22 @@ variables.
 | `SMTP_PASS`            |                            | account password                                                 |
 | `SMTP_AUTH_METHOD`     | `plain`                    | auth method to use with SMTP. Defaults to plain                  |
 | `PORT`                 | `3000`                     | port to run the server on defaults to 3000                       |
+
+## License
```
Comment (butlerx) [inline README.md]: this isnt needed as its covered by the license section of the package.json

Comment (GoldenBadger) [reply README.md]: I will edit the `package.json` but I think a more detailed clause is still appropriate here. Where else is the license mentioned?

**Pull request merged**